CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.40.105](https://github.com/brave/brave-browser/releases/tag/v1.40.105)
+
+ - Added Dapps support for Brave Wallet. ([#23159](https://github.com/brave/brave-browser/issues/23159))
+ - Added Non-Fungible Token support for Brave Wallet. ([#19565](https://github.com/brave/brave-browser/issues/19565))
+ - Added pre-loaded EVMs into Brave Wallet network selector (supported by 0x). ([#22888](https://github.com/brave/brave-browser/issues/22888))
```

Comment:
Remove added. Just pre loaded EVMs is fine

Reply:
Usually use a prefix in front of each note so lets just leave it as desktop using the same one which was reviewed by several people as well.